intermittent_failures.md

```diff
@@ -0,0 +1,138 @@
+
+The Automated Review (AR) pipeline performs builds and tests, as well as additional configuration and asset processing steps. Pull Requests are required to be verified by Automated Review before changes get merged. This Continuous Integration system also executes "branch update jobs" after merges, to detect complex merge-order issues which can occur in the time between verification and merge. A separate pipeline also executes at a slower cadence, which runs a wider array of periodic tests and build variants. [SIG-Build](https://github.com/o3de/sig-build/) owns the Jenkins service and scripts, and can advise on build failures. [SIG-Testing](https://github.com/o3de/sig-build/) owns test-tools as well as sanity-tests in this pipeline, which make sure the tools stay functional. While these SIGs maintain the foundation for testing, the majority of tests executed in this pipeline are owned by other SIGs. These tests, along with the applications they target, must operate deterministically.
+
+If you are blocked, you own fixing what blocks you and/or finding support. Even if you are not blocked, the community will appreciate fixes or feedback submitted to the appropriate SIG. Which SIG owns a failure is not always obvious, as the Automated Review pipeline executes code maintained by every SIG in O3DE. If you have questions, reach out over [Discord](https://discord.gg/p3padwr58u)!
+
+## Policy
+
+No intermittently failing test or feature is allowed, with extremely limited exceptions. If you see a failure that appears nondeterministic, immediately remove it following these steps:
+
+1. Determine if the failure is intermittent
+   - If possible, execute on your local machine multiple times to tally a failure rate, and attempt to debug the issue
+   - Execute the Jenkins pipeline as many as 10 times, and save at least one text file with a failing log
+   - If the failure occurs consistently ***DO NOT CONTINUE*** following these steps, and instead reach out over [Discord](https://discord.gg/p3padwr58u)
+2. [Cut a GitHub Issue](https://github.com/o3de/o3de/issues/new/choose) to track both fixing the underlying issue and re-enabling the test
+   - Add a label for the SIG which maintains that feature 'sig/XYZ' ***OR*** use label 'needs-sig'
+3. The source of intermittent failure should be immediately removed from the pipeline
+   - If possible, submit a new pull request which reverts or disables the intermittent module, feature, test, or hardware-configuration
+
+### Exceptions
+
+The policy above applies to all code, scripts, and hardware used in the Automated Review Pipeline with the following exceptions:
+
+- It is rarely more risky to remove an unstable critical test (or feature) than to leave it in a known intermittently failing state
+  - If you have a reason to not follow this policy, please reach out to SIG-Testing on [Discord](https://discord.gg/p3padwr58u)!
```

Comment:
indentation on this line seems off

Reply:
This is intentionally on a sub-bullet point, the two extra spaces will indent its layout